.github/workflows/conformance-kubespray.yaml

```diff
@@ -0,0 +1,251 @@
+name: Conformance Kubespray (ci-kubespray)
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  workflow_dispatch:
+    inputs:
+      PR-number:
+        description: "Pull request number."
+        required: true
+      context-ref:
+        description: "Context in which the workflow runs. If PR is from a fork, will be the PR target branch (general case). If PR is NOT from a fork, will be the PR branch itself (this allows committers to test changes to workflows directly from PRs)."
+        required: true
+      SHA:
+        description: "SHA under test (head of the PR branch)."
+        required: true
+      extra-args:
+        description: "[JSON object] Arbitrary arguments passed from the trigger comment via regex capture group. Parse with 'fromJson(inputs.extra-args).argName' in workflow."
+        required: false
+        default: '{}'
+  push:
+    branches:
+      - 'main'
+  schedule:
+    - cron: '0 0 * * 0'
+
```

Comment:
Were you able to test the new version of the workflow? Maybe it'd help to temporarily add a commit for `pull_request: {}` trigger so we can see that the workflow runs correctly. Arguably such a trigger could stay in with a path filter for this workflow path just to simplify subsequent pull requests for this file.